packages/knip/src/plugins/node-modules-inspector/index.ts

```diff
@@ -0,0 +1,26 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { toUnconfig } from '../../util/plugin-config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/antfu/node-modules-inspector
+
+const title = 'node-modules-inspector';
+
+const enablers = ['node-modules-inspector'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config: string[] = [...toUnconfig('node-modules-inspector.config')];
+
+const args = {
+  binaries: ['node-modules-inspector'],
+  config: true,
```

Comment:
This doesn't seem like a documented feature?

- If it is a feature, then would be good to have the fixture point to a JS/TS file for coverage (so the file isn't unused)
- If it's not actually a feature we can remove `args` here as this `binaries` value is the default.

Reply:
Huh, I thought it was documented. It does support it though:
https://github.com/antfu/node-modules-inspector/blob/86a56423d3210ffe412c81e72dd426516a8aa832/packages/node-modules-inspector/src/node/cli.ts#L82

Reply:
Great!